tomviz/Operator.h

```diff
@@ -53,6 +53,7 @@ class Operator : public QObject
   virtual bool deserialize(const pugi::xml_node& ns)=0;
 
   virtual EditOperatorWidget* getEditorContents(QWidget* parent) = 0;
+  virtual bool hasCustomUI() const = 0;
```

Comment:
This makes tomviz unable to be built because it conflicts with #320 , in which the "TranslateAlignOperator.h" file does not override the new virtual function.